lib/authtrail.rb

```diff
@@ -4,14 +4,16 @@
 
 # modules
 require "auth_trail/engine"
+require "auth_trail/idenitity_lookup"
 require "auth_trail/manager"
 require "auth_trail/version"
 
 module AuthTrail
   class << self
-    attr_accessor :exclude_method, :geocode, :track_method
+    attr_accessor :exclude_method, :geocode, :track_method, :identity
   end
   self.geocode = true
+  self.identity = AuthTrail::IdentityLookup.new(:email)
```

Comment:
I'm not sure we get anything from an extra class. I think we can just add it here.

Reply:
I don't like having lambda's in code unless they givie a specific advantage -> which is the reason I went with the class.

Also the way I have written this allows for me to easily extend it for the use-case I have:
```
Authtrail.indentity = AuthTrail::IdentityLookup.new(:username)
```
which makes me happy.

Ultimately I guess it comes down to code style - I prefer this way - if you still prefer the other let me know and I will update the PR. 😄

Reply:
I think a class makes it less clear how to customize it fully. Two of the other settings currently use proc/lambda - `exclude_method` and `track_method` - so I think it makes sense from a consistency perspective as well. We also probably want to name it `identity_method`.

Reply:
done